app/Http/Controllers/Tabbar/Navigation/Dev/DevController.php

```diff
@@ -3,30 +3,21 @@
 namespace App\Http\Controllers\Tabbar\Navigation\Dev;
 
 use App\Http\Controllers\Controller;
-use App\Settings\PathInfo;
-use Parsedown;
 
-class DevController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | DevController
-    |--------------------------------------------------------------------------
-    |
-    | Devコントローラ
-    |
-    */
+/**
+ * DevController
+ * 
+ * Devタブ内で表示するコンテンツを扱う
+ */
+class DevController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition